### ISSUE TITLE
Color Contrast: Tweet

### ISSUE THREAD

**Issue by AutoSponge** [Hashnode/user-feedback#174]
**Describe the bug**
Element has insufficient color contrast of 2.82 (foreground color: #ffffff, background color: #1da1f2, font size: 12.0pt (16px), font weight: normal). 

**To Reproduce**
Element location: `.twitter > span`
Element source: `<span>Tweet</span>`

**Expected behavior**
Expected contrast ratio of 4.5:1

Possible fix:
- Switch background-color to Twitters `:hover` color: `#1A91DA` for 3.43:1.

**Comment by BolajiAyodeji** [comment]
Hi @AutoSponge,

Thanks for the amazing feedback. We are working on getting this fixed and we'll update you soon.
Keep the feedbacks coming :)
